proxyserver/src/main/java/com/lyft/data/proxyserver/ProxyHandler.java

```diff
@@ -68,7 +68,11 @@ protected void debugLogHeaders(HttpServletRequest request) {
       Enumeration<String> headers = request.getHeaderNames();
       while (headers.hasMoreElements()) {
```

Comment:
@cmiao we would potentially take the chance here to optimize this a little bit by only iterating if debug level is enabled, wdyt?

Reply:
It's optimised this way a few lines above :)

Also seems only this debug log is used, we can optimize others in next refactor.